pkg/storage/replica.go

```diff
@@ -3767,9 +3769,17 @@ func (r *Replica) tick() (bool, error) {
 		// difficult to completely eliminate them so we want to minimize the
 		// disruption when they do occur.
 		//
-		// For more details, see #9372.
-		// TODO(bdarnell): remove this once we have fully switched to PreVote
-		if !enablePreVote {
+		// Without CheckQuorum, the call to TickQuiesced is less critical,
+		// but still improves our responsiveness to node failures by
+		// priming the replica to start an election immediately upon
```

Comment:
> priming the replica to start an election immediately upon unquiescing

Priming any replica or priming followers? Is this important for a leader?

Reply:
With CheckQuorum off, this only affects followers.